### PR TITLE
Symfony 3.0 compatibility 

### PR DESCRIPTION
**Pull request by gepo**
Fix service injection definition according https://github.com/symfony/symfony/pull/16285

### CODE REVIEW

File: Module/JobClass.php [hunk 1]
```diff
@@ -25,8 +25,10 @@
  *
  * @since 2.3.1
  */
-class JobClass extends ContainerAware
+class JobClass
```
Comment (Tobion) [inline Module/JobClass.php]: I assume you need to implement the containerinterface

File: Module/JobClass.php [hunk 2]
```diff
@@ -25,8 +25,10 @@
...
 {
+    use ContainerAwareTrait;
```
Comment (Tobion) [inline Module/JobClass.php]: this increases the min php version in composer.json

Comment (gepo) [inline Module/JobClass.php]: fix it

**Comment by daum** [comment]
@gepo Sorry for the delay on this.  I think we're ready to get this merged in, however your composer.json still appears to be referencing the 2.X components if I'm not mistaken?  Do you mind updating that to be in line with 3.X components, and then we can merge this.

**Comment by daum** [comment]
@gepo actually I updated a PR( https://github.com/mmoreram/GearmanBundle/pull/168 ) to be fully functional.  It has a few more updates to the composer/testing suite.  Assuming that is functional, I'll merge it in today and tag our Symfony 3.X branch with v4.0 tags as the 3.X tags will refer to symfony 2.

**Comment by daum** [comment]
Going to close this, #171 brought this to Symfony 3.0 compatibility.

**Pull request closed without merging**